sdk/storage/azure-storage-common/inc/azure/storage/common/paged_response.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <cstdlib>
+#include <string>
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/raw_response.hpp>
+
+namespace Azure {
+
+template <class Derived> class PagedResponse {
```

Comment:
Add this in the `New Features` section of https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/azure-storage-common/CHANGELOG.md

- Added `Azure::PagedResponse<T>` for returning paginated collections from client APIs.

Reply:
done in https://github.com/Azure/azure-sdk-for-cpp/pull/2129